### ISSUE TITLE
manual changes are lost when importing/exporting theme

### ISSUE THREAD

**Issue by ceblanda** [jquery/themeroller.jquerymobile.com#208]
I created a theme and downloaded it, then I opened the css file and made some changes to things I couldn't though the interface (such as a:visited).  Then I uploaded the theme back into ThemeRoller so I could use the interface to make additional changes.  When I downloaded it again, the manual changes I had made were lost.

**Comment by agcolom** [comment]
I've just checked and was able to create a theme, download it, manually made changed (to a:hover and button hover colors), and reimported into the interface without any issue and the manual changes were present in the interface. I'm going to close this issue as I am not able to reproduce and we didn't receive details of what was not working. 
@ceblanda If this is still an issue for you, please give us all the details of what manual changes are not imported into the interface, and I will re-open the issue.

**Comment by ceblanda** [comment]
When I tested, if I added additional items they were lost when imported and exported.
--C

**Comment by agcolom** [comment]
@ceblanda thanks very much for your reply. I am not sure themeroller should take manual additions (as opposed to manual editions of provided values), as I believe additions should go into a separate css file added after the themeroller css, as suggested by @sogen. 
@arschmitz @gabrielschulhof @jaspermdegroot could you please confirm whether this is the case, thank you.

**Comment by ceblanda** [comment]
Unfortunately that will not work in my situation.
--C

**Comment by jaspermdegroot** [comment]
Making changes in the CSS file shouldn't be a problem as long as all the property/value pairs where the value has a comment in it (`/*{name-of-the-setting}*/}`) stay.

**Comment by agcolom** [comment]
I've tried by adding ```.myclass { background-color: rgb(255,0,0) /*{this is for testing purposes}*/;}``` in various places but this was lost in the process. So is this a bug? (reopening the issue until we know for sure).

**Comment by jaspermdegroot** [comment]
And without the `/*{this is for testing purposes}*/`, is the custom rule removed?
That `/*{ ... }*/` specific comment format shouldn't be used in anything that you add to the file. What I meant was that the original lines that have a comment of that format in it shouldn't be removed.

**Comment by agcolom** [comment]
@jaspermdegroot yes, in both cases (with comments and without comments) the addition is lost
